File: Python3/1753_maximum_score_from_removing_stones.py
# https://leetcode.cn/problems/maximum-score-from-removing-stones/

class Solution:
    def maximumScore(self, a, b, c):
        """
        Inputs:
            a, b, c: int
        Outputs:
            res: int
        """
        s = a + b + c
        max_ = max(a, b, c)
        if s <= max_ + max_:
            res = s - max_
        else:
            res = (s - max_ - max_) // 2 + max_
        return res

File: Cpp/1753_maximum_score_from_removing_stones.cpp
// https://leetcode.cn/problems/maximum-score-from-removing-stones/

class Solution {
public:
    int maximumScore(int a, int b, int c) {
        int s = a + b + c;
        int max_ = max({a, b, c});
        int res = 0;
        if (s <= max_ + max_) {
            res = s - max_;
        }
        else {
            res = (s - max_ - max_) / 2 + max_;
        }
        return res;
    }
};
